examples/daal/cpp/source/compression/compression_online.cpp

```diff
@@ -58,7 +58,7 @@ int main(int argc, char *argv[]) {
     prepareMemory();
 
     /* Create a compressor */
-    Compressor<zlib> compressor;
+    Compressor<lzo> compressor;
```

Comment:
I guess same should be done for java examples

Reply:
suggest to do this in separate PR.  this one intended for cmake and cpp examples were changed to remove warnings.